benchmarks/splines.cpp

```diff
@@ -148,7 +148,8 @@ static void characteristics_advection(benchmark::State& state)
         ddc::parallel_for_each(
                 feet_coords.domain(),
                 KOKKOS_LAMBDA(ddc::DiscreteElement<DDimX, DDimY> const e) {
-                    feet_coords(e) = ddc::coordinate(ddc::select<DDimX>(e)) - ddc::Coordinate<X>(0.0176429863);
+                    feet_coords(e) = ddc::coordinate(ddc::select<DDimX>(e))
+                                     - ddc::Coordinate<X>(0.0176429863);
```

Comment:
weird

Reply:
could be that the CI does not check this file ?

Reply:
https://github.com/CExA-project/ddc/blob/0da9a03f43e761848c57b7536f7217730a54a323/.github/workflows/tests.yml#L19 seems to not be checked. So maybe a different version of clang-format has been used leading to a different formatting

Reply:
#547